agent/consul/state/acl.go

```diff
@@ -563,6 +563,7 @@ func fixupRolePolicyLinks(tx ReadTxn, original *structs.ACLRole) (*structs.ACLRo
 }
 
 // ACLTokenSet is used to insert an ACL rule into the state store.
+// Deprecated (ACL-Legacy-Compat)
 func (s *Store) ACLTokenSet(idx uint64, token *structs.ACLToken, legacy bool) error {
```

Comment:
A+ for keeping these state store functions, since they'll be necessary to replay raft logs created from prior to the upgrade if someone was still using legacy operations right up until the 1.11 upgrade.